administrator/components/com_content/models/shared.php

```diff
@@ -0,0 +1,218 @@
+<?php
+/**
+ * @package     Joomla.Administrator
+ * @subpackage  com_content
+ *
+ * @copyright   Copyright (C) 2005 - 2016 Open Source Matters, Inc. All rights reserved.
+ * @license     GNU General Public License version 2 or later; see LICENSE.txt
+ */
+defined('_JEXEC') or die;
+/**
+ * About Page Model
+ *
+ * @since  _DEPLOY_VERSION_
+ */
+class ContentModelShared extends JModelList
+{
+	/**
+	 * Constructor.
+	 *
+	 * @param   array  $config  An optional associative array of configuration settings.
+	 *
+	 * @see     JController
+	 * @since    _DEPLOY_VERSION_
+	 */
+	public function __construct($config = array())
+	{
+		if (empty($config['filter_fields']))
+		{
+			$config['filter_fields'] = array(
+				'id', 'a.id',
+				'title', 'a.title',
+				'alias', 'a.alias',
+				'checked_out', 'a.checked_out',
+				'checked_out_time', 'a.checked_out_time',
+				'catid', 'a.catid', 'category_title',
+				'state', 'a.state',
+				'access', 'a.access', 'access_level',
+				'created', 'a.created',
+				'created_by', 'a.created_by',
+				'created_by_alias', 'a.created_by_alias',
+				'ordering', 'a.ordering',
+				'featured', 'a.featured',
+				'share','a.share',
```

Comment:
see https://github.com/joomla-projects/gsoc16_shareable-draft-content/pull/10#issuecomment-242520205 this line may should be removed.